### PR TITLE
Storybook/contact info

### PR DESCRIPTION
**Pull request by MAlngaawy**

### CODE REVIEW

File: src/stories/ContactInfo.stories.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        'https://charity-cms-dev.s3.eu-central-1.amazonaws.com/phone-iconpng_11d0269061.png'
+    },
+    url: 'tel: 666 888 6666'
+  },
+  {
+    _id: '5ea86143fd37db00172c7402',
+    title: '66 Broklyn Street USA',
+    sub_title: 'Visit us',
+    icon: {
+      _id: '5eb08d345f16e8001708925b',
```
Comment (Nilegfx) [inline src/stories/ContactInfo.stories.js:65]: you don't need `_id` here

File: src/stories/ContactInfo.stories.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import { ContactInfo } from '../components/ContactInfo';
+
+export default {
+  title: 'ContactInfo',
+  component: ContactInfo,
+  decorators: [withKnobs]
+};
+
+// Data
+const socialData = [
```
Comment (Nilegfx) [inline src/stories/ContactInfo.stories.js:12]: please remove all unnecessary keys from social and contact mocked data objects, things like (`_id` .. etc)

File: src/components/ContactInfo/index.jsx [hunk 3]
```diff
@@ -3,10 +3,11 @@ import cn from 'classnames';
 import { charityAPI } from '../../clients';
```
Comment (Nilegfx) [inline src/components/ContactInfo/index.jsx:3]: Please use `useCharityAPI` hook. 
even if you have two API calls for this component, you still can use the `useCharityAPI` hooks twice ;)

File: src/components/ContactInfo/index.jsx [hunk 4]
```diff
@@ -1,103 +1,192 @@
...
+    height={50}
+    viewBox="0 0 100 100"
+    backgroundColor="#f5f5f5"
+    foregroundColor="#f5f5f5"
+  >
+    <circle cx="40" cy="40" r="33" />
+  </ContentLoader>
+);
+
+const ContactLoader = () => (
```
Comment (Nilegfx) [inline src/components/ContactInfo/index.jsx:179]: How about something like this?
```js
const ContactLoader = ({count}) => {
return Array(count).fill(1).map(()=>(<ContentLoader
    speed={2}
    width={200}
    height={60}
    viewBox="0 0 200 60"
    backgroundColor="#f5f5f5"
    foregroundColor="#f5f5f5"
  >
    <circle cx="46" cy="26" r="21" />
    <rect x="74" y="10" rx="0" ry="0" width="163" height="8" />
    <rect x="80" y="30" rx="0" ry="0" width="94" height="8" />
  </ContentLoader>))
}
```

and using it like this
```js
<ContactLoader count=3 />
```

Comment (MAlngaawy) [reply src/components/ContactInfo/index.jsx:179]: I Like the way you mentee us 😃

**Pull request merged**